plugins/Rockets/jsonSerialization.h

```diff
@@ -546,6 +546,7 @@ inline void init(brayns::Scene* s, ObjectHandler* h)
 inline void init(brayns::ApplicationParameters* a, ObjectHandler* h)
 {
     h->add_property("engine", &a->_engine, Flags::IgnoreRead | Flags::Optional);
+    h->add_property("plugins", &a->_plugins, Flags::Optional);
```

Comment:
Here I think it would be appropriate to bitmask the flag Flags::IgnoreRead so that the clients cannot modify the list of plugins.

Reply:
Added 6ecc17e